app/services/platform-apps/api/index.ts

```diff
@@ -74,30 +75,37 @@ export class PlatformAppsApi {
         if (!authorized) break;
       }
 
-      ((this.modules[moduleName].constructor as typeof Module).apiMethods || []).forEach(methodName => {
-        api[moduleName][methodName] = async (...args: any[]) => {
+      ((this.modules[moduleName].constructor as typeof Module).apiMethods || []).forEach(
+        methodName => {
+          api[moduleName][methodName] = async (...args: any[]) => {
+            if (authorized) {
+              return await this.modules[moduleName][methodName](context, ...args);
+            } else {
+              throw new Error(
+                'This app does not have permission to access this API. ' +
+                  `Required permissions: ${this.modules[moduleName].permissions}`,
+              );
+            }
+          };
+        },
+      );
+
+      ((this.modules[moduleName].constructor as typeof Module).apiEvents || []).forEach(
+        eventName => {
           if (authorized) {
-            return await this.modules[moduleName][methodName](context, ...args);
+            api[moduleName][eventName] = this.modules[moduleName][eventName];
           } else {
-            throw new Error('This app does not have permission to access this API. ' +
-              `Required permissions: ${this.modules[moduleName].permissions}`);
+            api[moduleName][eventName] = async () => {
+              throw new Error(
+                'This app does not have permission to access this API. ' +
+                  `Required permissions: ${this.modules[moduleName].permissions}`,
+              );
+            };
           }
-        };
-      });
-
-      ((this.modules[moduleName].constructor as typeof Module).apiEvents || []).forEach(eventName => {
-        if (authorized) {
-          api[moduleName][eventName] = this.modules[moduleName][eventName];
-        } else {
-          api[moduleName][eventName] = async () => {
-            throw new Error('This app does not have permission to access this API. ' +
-              `Required permissions: ${this.modules[moduleName].permissions}`);
-          }
-        }
-      });
+        },
+      );
```

Comment:
All of this looks like just a prettier pass.  Just want to make sure you didn't change any logic here, right?

Reply:
We did not.